test/tests.cpp

```diff
@@ -496,6 +496,113 @@ TEST(SetAndGetWordTests, NegDoubleDataWord)
     EXPECT_EQ(exp_parity, wordWithLabel.get<parity>());
 }
 
+TEST(CustomizationTests, Set)
```

Comment:
need better naming for these tests: what are you testing. Example `customized_setter`